docs/IE_DG/ONNX_Support.md

```diff
@@ -9,10 +9,34 @@ InferenceEngine::Core core;
 auto network = core.ReadNetwork("model.onnx");
 ```
 
+**Reshape feature:**
+
 OpenVINO™ doesn't provide a mechanism to specify pre-processing (like mean values subtraction, reverse input channels) for the ONNX format.
 If an ONNX model contains dynamic shapes for input, please use the `CNNNetwork::reshape` method for shape specialization.
 
-Unsupported types of tensors:
+**Weights saved in external files:**
+
+OpenVINO™ supports ONNX models which use weights saved in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
+If you want to read such model you should use ReadNetwork overload which takes `modelPath` as input parameter (both `std::string` and `std::wstring`).
```

Comment:
Should weightPath be empty? What happens if user specifies it?

Reply:
Right, `weightPath` should be empty.
If user fills it with sth, there are two option:
1. If pass is not correct path the exception is thrown from [ie_network_reader.cpp](https://github.com/openvinotoolkit/openvino/blob/6c341df85a3016222b9d16f02d766c28cb22924d/inference-engine/src/inference_engine/ie_network_reader.cpp#L211)
2. If pass is correct the exception is thrown from [ie_onnx_reader.hpp](https://github.com/openvinotoolkit/openvino/blob/3c718809d3d124e5156d240d660de15e7fd857fc/inference-engine/src/readers/onnx_reader/ie_onnx_reader.hpp#L39)

Reply:
I added update about not supported `ReadNetwork` inputs

Reply:
Please add information that `weightPath` must be empty. In other case you will get an exception.

Reply:
@ilyachur I added this information (+ about other not supported parameters) below:
```
The described mechanism is the only possibility to read weights from external files. The following input parameters of `ReadNetwork` function overloads are NOT supported for ONNX models and should be passed as empty (or `nullptr`)
...
```
just to not mixing topics.
Do you think that it will be clear?

Reply:
+ you are right that I'll add info about exception